students/14th/team_1/seungje/project_westagram/account/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class Account(models.Model):
+
+    user_name = models.CharField(max_length=40,default='')
+    user_number = models.CharField(max_length=40,default='')
+    user_email = models.CharField(max_length=40,default='')
+
+    user_password = models.CharField(max_length=40,default='')
```

Comment:
```suggestion
    user_name = models.CharField(max_length=40,default='')
    user_number = models.CharField(max_length=40,default='')
    user_email = models.CharField(max_length=40,default='')
    user_password = models.CharField(max_length=40,default='')
```
`=`을 기준으로 코드 간격 정렬해주세요 :) 
password 또한 Account의 속성 가운데 하나이므로 간격 분리 하지 않으셔도 됩니다.